### ISSUE TITLE
[Rule] use-implicit-booleaness-not-comparison 

### ISSUE THREAD

**Issue by bhirsz** [MarketSquare/robotframework-robocop#891]
Some condition expression can be simplified, ie:

1.
```
IF    ${status} == ${TRUE}
```

to

```
IF    ${status}
```

2.
```
IF    ${status} != ${TRUE}
```

to

```
IF    not ${status}
```

**Comment by bhirsz** [comment]
I have came up with similar rules but I had problem with naming. After research I have noticed that Google Python code guidelines and pylint already had rules for it. I will use their naming for this and following rules for expressions.

**Comment by bhirsz** [comment]
It should also detect

```
IF ${var} == [] 
```
And similar
